ajax/libs/convnetjs/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "convnetjs",
+  "filename": "convnet-min.js",
+  "version": "0.3.0",
+  "description": "Deep Learning in Javascript. Train Convolutional Neural Networks (or ordinary ones) in your browser.",
+  "keywords": [
+    "machine",
+    "learning",
+    "AI",
+    "convnet"
```

Comment:
These seem from `bower.json`. Did you confirm is if these keywords are better than those in https://unpkg.com/convnetjs@0.3.0/package.json?